### DISCUSSION TITLE
Not worked for me in update

### DISCUSSION THREAD

**Discussion by abdosaeedelhassan** [spatie/laravel-activitylog#740] [Q&A] [answered]
i do all installation steps as configrations steps as described in next vedio
https://www.loom.com/share/a62faa1fb33e4e1e96bdd480972e6591

but inside model not worked for me in update, only on create

**Comment by abdosaeedelhassan** [comment]
@jamieshiers

**Reply by jamieshiers** [reply]
@abdosaeedelhassan The only thing I can think of afer watching your video is that the model isn't being updated, when your submitting the form. Have you tried changing a value on your form to see if that logs the activity?

**Reply by abdosaeedelhassan** [reply]
yes, in database the fields are changed, also  i am tried with a fresh copy of laravel also worked for me in creation only.

**Comment by abdosaeedelhassan** [comment, marked as answer]
finally i found the solution

not worked like
`Orders::where('id', '=', $orders->id)->update($data);`

worked like
`Orders::find($orders->id)->update($data);`

i don't know why where condition not fired the event

**Reply by Gummibeer** [reply]
It's not the `where()` but the `find()` your first example runs the update query without retrieving the model. It immediately generates an update query with the where condition.
Because we can't monitor the DB but only the model this triggers no event.

If you call `find()`, `first()`, `get()` or any other method that runs a select query and returns eloquent model(s) the update call will run on the model instead of the query builder and therefore can trigger the eloquent events.

There's also a `whereKey()` query method you can use.

PS: models should be named singular.
